tests-e2e/features/assets/digital-twins/digital-twin-detail-permissions.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Digital Twin - Detail View for missing permissions
+
+  Background:
+    Given I log in to evan.network using vue
+
+  Scenario: Access foreign twin without permissions
+    When I am on the path "#/dashboard.vue.evan/detail.digital-twin.evan/0x56fc9d62fce20193cd0c1d597aa7919630895b0b"
+      Then I want to see a text including "Test Car Twin"
```

Comment:
Indentation should be the same for When and Then but "But" and "And" should be indented.

Reply:
I don't think there is a concrete rule for this?
Official doc doesn't use any at all: https://cucumber.io/docs/gherkin/reference/
Let's not ride the dead horse longer than necessary? ;)